blueocean-dashboard/package.json

```diff
@@ -41,6 +41,7 @@
     "gulp-mocha": "3.0.1",
     "mocha": "3.1.0",
     "nock": "8.0.0",
+    "parse-link-header": "1.0.1",
```

Comment:
looks like a small dep - ok for you @sophistifunk ?

Reply:
Yeah I checked it out it's focused and only has one other small dependency.